src/Config/ObjectTypeDefinition.php

```diff
@@ -67,6 +73,29 @@ private function treatFieldsDefaultAccess(ArrayNodeDefinition $node): void
         ->end();
     }
 
+    /**
+     * set empty fields.accessConfig with fieldsDefaultAccessConfig values if is set?
+     */
+    private function treatFieldsDefaultAccessConfig(ArrayNodeDefinition $node): void
+    {
+        $node->validate()
+            ->ifTrue(function ($v) {
+                return array_key_exists('fieldsDefaultAccessConfig', $v) && null !== $v['fieldsDefaultAccessConfig'];
```

Comment:
```suggestion
                return isset($v['fieldsDefaultAccessConfig']);
```
The isset operator checks, if the key exists and is not null